sdk/core/azure-core/inc/azure/core/http/curl/curl_session.hpp

```diff
@@ -319,17 +319,28 @@ namespace Azure { namespace Core { namespace Http {
       return eof && m_sessionState != SessionState::PERFORM;
     }
 
+    /**
+     * @brief All connection will be open and requested to maintain the channel open to re-use the
+     * connection.
+     *
+     * @remark This option can be disabled from the transport adapter options. When disabled, the
+     * session won't return connections to the connection pool. Connection will be clean as soon as
```

Comment:
I don't know what `clean` means. `destroyed`? `closed`?
```suggestion
     * session won't return connections to the connection pool. Connection will be closed as soon as
```